etc/bento.env

```diff
@@ -439,3 +439,19 @@ BENTO_CBIOPORTAL_SESSION_DATABASE_IMAGE_VERSION=6.0.7
 BENTO_CBIOPORTAL_SESSION_DATABASE_CONTAINER_NAME=${BENTOV2_PREFIX}-cbioportal-session-db
 BENTO_CBIOPORTAL_SESSION_DATABASE_DATA_DIR=${BENTO_FAST_DATA_DIR}/cbioportal/session_db
 # Uses BENTO_CBIOPORTAL_SESSION_NETWORK as the network
+
+# Monitoring
+BENTO_MONITORING_NETWORK=${BENTOV2_PREFIX}-monitoring-net
+BENTOV2_PRIVATE_GRAFANA_URL=${BENTOV2_PORTAL_PUBLIC_URL}/api/grafana
+BENTOV2_PUBLIC_GRAFANA_URL=${BENTOV2_PORTAL_PUBLIC_URL}/grafana
+BENTOV2_LOKI_BASE_IMAGE=grafana/loki
```

Comment:
are these base images, or just images? I don't see anywhere where you use this as a base image for building another image.

Reply:
yeah you're right just images